public/js/modal.js

```diff
@@ -0,0 +1,27 @@
+﻿var modal = document.getElementById('myModal');
+var cancelbtn = document.getElementsByClassName('canceldelete')[0];
+// Get the button that opens the modal
+
+// Get the <span> element that closes the modal
+var span = document.getElementsByClassName("close")[0];
+
+// When the user clicks on the button, open the modal
+
+
+// When the user clicks on <span> (x), close the modal
+span.onclick = function () {
+    modal.style.display = "none";
+}
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function (event) {
+    if (event.target == modal) {

```

Comment:
Use triple equal here. Always try to satisfy the linter as much as you can.

Reply:
done